.travis.yml

```diff
@@ -23,6 +23,7 @@ include:
 sudo: required
 
 script:
+  - chmod +x ./build.sh
```

Comment:
its already in +x

Reply:
@xeon-zolt But yesterday, it gave me an error os i added this